Update lockscreen widgets with On-Call (#32287)
DataDog/documentation: content/en/mobile/guide/configure-mobile-device-for-on-call.md

<a href="https://datadog-on-call.s3.amazonaws.com/datadog-on-call.vcf">Download the current version of the Datadog On-Call contact card</a>. <strong>Note</strong>: The contact card is subject to change at any time.
</div>

> ## On-Call Mobile Widgets

Reviewer: ```suggestion
## On-Call mobile widgets
```

> Add On-Call homescreen and lockscreen widgets to easily access your pages and shifts.

Reviewer: ```suggestion
Add On-Call homescreen and lockscreen widgets to access your pages and shifts.
```

> ### On-Call homescreen widget

Reviewer: ```suggestion
### On-Call home screen widget
```

View your On-Call shifts and On-Call pages on your mobile home screen with Datadog widgets.

You can customize your On-Call shifts widgets by filtering on:

- Organization

> - Timeframe period

Reviewer: ```suggestion
- Period of time
```

You can customize your On-Call pages widgets by filtering on:

- Organization
- Team
- Order

**Note**: You can add additional filters for the On-Call pages widget.

#### Edit an On-Call shift widget

{{< tabs >}}
{{% tab "iOS" %}}

{{< img src="service_management/mobile/ios_shifts_widget_may_2025.png" alt="Configured home screen on-call shift widgets displayed on iOS screens" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

1. Long press on the widget to configure.

> 2. Tap **Edit Widget** to bring up the configure screen.

Reviewer: ```suggestion
2. Tap **Edit Widget** to bring up the configuration screen.
```

3. Select the **Organization** and **Period** you would like to see your On-Call shifts.
4. Tap out of the widget to validate your selection and exit the configuration screen.

{{% /tab %}}
{{% tab "Android" %}}

{{< img src="service_management/mobile/android_shifts_widget_may_2025.png" alt="Configured home screen On-Call shift widgets displayed on Android screens" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}

1. Tap on the widget to configure.

> 2. Select the **Organization** and **Time Period** you would like to see your On-Call shifts.

Reviewer: ```suggestion
2. Select the **Organization** and **Time Period** you would like to see your On-Call shifts for.
```

3. Tap **✓** to save the configuration.
4. Long press and resize the widget to fit your preference.

{{% /tab %}}
{{< /tabs >}}

### On-Call lockscreen widget

> On-Call lock screen widget displays your On-Call status. Lockscreen widgets are only available on iOS.

Reviewer: ```suggestion
The On-Call lockscreen widget displays your On-Call status. Lockscreen widgets are only available on iOS.
```
